src/main/java/com/kodilla/ecommercee/dto/UserDto.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.ecommercee.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+public class UserDto {
+
+    private Long id;
+    private String username;
+    private Integer status;
```

Comment:
Status uzytkownika (tylko zgaduje) zapewne jest online lub offline, wtedy mozna byloby uzyc booleana zamiast Integera, ale jesli projekt zaklada, ze te pole ma byc Integer, to ok. Innych uwag do tego kodu nie posiadam.

Reply:
Do przechowywania statusów polecam użyć enuma

Reply:
Też o tym myślałem, ale jesteśmy związani specyfikacją - zwracany JSON musi być identyczny, jak wyspecyfikowany przykład - w przykładzie pole status ma wartość liczbową.

Reply:
Nie jest to problemem: https://stackoverflow.com/a/37833901 
https://stackoverflow.com/a/56285491
ale pozostawiam wg uznania :)